architecture.tex

```diff
@@ -232,13 +232,13 @@ \subsection{Trust Management}\label{sec-trust}
 established.
 
 Additionally, Seattle uses asymmetrical cryptography as
-a technical means to map out trust:
-Its core computational component, the sandbox, has a notion
-of an \textit{admin}
+a technical means to map out trust.
+Its core computational component, the sandbox, incorporates the rolesof
```

Comment:
(Fixed a typo in "rolesof", then merged.)

Reply:
Yeah...thought I had fixed that. Glad you caught it.